Web/HTML/html5-input-form-attributes/formnovalidate-and-novalidate.md

```diff
@@ -41,12 +61,32 @@ Complete the code by adding the appropriate input form attributes to disable the
 
 Complete the code by adding the appropriate input form attributes to disable the validation within the form element and in the submit button.
 
-```<form action="/action_page.php" ??? >
- E-mail: <input type="email" name="user_email">
+```
+<form 
+ action="/action_page.php" ??? >
+ E-mail: 
+ <input type="email" 
+  name="user_email">
  <input type="submit" ??? >
-</form>```
-
+</form>
+```
 * novalidate 
 * formnovalidate
 * validate
 
+
+---
+## Quiz
+
+### How much do you know about HTML form input attributes?
+
+Which statement is not true regarding the `novalidate` attribute?
```

Comment:
is true? or isn't true?

Reply:
Isn't true, because when both are present the formnovalite will override the novalidate(3rd answer), not the other way around

* The `novalidate` attribute overrides the `formnovalidate` attribute.
  * The HTML `novalidate` form input attribute is used to specify that the input should not be validated when submitted.
  * When specified, the `formnovalidate` attribute overrides the `novalidate` attribute.
  * It can be used with the `type"submit"` and `type"image"` attributes.